lib/demos/reference/motion_demo_container_transition.dart

```diff
@@ -0,0 +1,540 @@
+                    return _ExampleSingleTile(openContainer: openContainer);
+                  },
+                ),
+                const SizedBox(height: 16.0),
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                    const SizedBox(width: 8.0),
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 16.0),
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionSmallPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                    const SizedBox(width: 8.0),
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
+                          return _SmallerCard(
+                            openContainer: openContainer,
+                            subtitle: GalleryLocalizations.of(context)
+                                .demoMotionSmallPlaceholderSubtitle,
+                          );
+                        },
+                      ),
+                    ),
+                    const SizedBox(width: 8.0),
+                    Expanded(
+                      child: _OpenContainerWrapper(
+                        transitionType: _transitionType,
+                        closedBuilder: (context, openContainer) {
```

Comment:
The closedBuilder looks to be the same for all _OpenContainerWrappers. If it is, it can be factored to be within the _OpenContainerWidget so you do not have to pass it here